resources/js/Components/AuthWidget.vue

```diff
@@ -1,11 +1,13 @@
 <template>
-    <div v-if="user" class="auth">
-        <!-- Stuff to show when user is logged in -->
-        <p>Hello, {{user.name}}!</p>
-    </div>
-    <div v-else class="guest">
-        <!-- Stuff to Show otherwise -->
-        <p>Hello, Guest!</p>
+    <div>
+        <div v-if="user">
+            <a :href="`https://www.wikidata.org/wiki/User:${user.name}`">
```

Comment:
I wonder if this link should open in a new tab?

Reply:
I asked @MattiaCapozziWMDE previously for the store and he said it shouldn't but will ask again.

Reply:
Leaving as it is for now, after reading @MattiaCapozziWMDE 's input.